### DISCUSSION TITLE
BIDS export

### DISCUSSION THREAD

**Discussion by jacobsen-noelle** [bids-standard/bids-specification#1420] [Ideas]
The bids function gives you the option to change your output directory; however, if your directory is not an empty folder, it will delete all folder contents. Despite the pop-up warning about file deletion (I thought it would just delete things in the "BIDS" folder, not parent directory"), I accidentally deleted a lot of files on my computer. Another lab member accidentally did the same. The function also overwrites permissions. This could potentially be catastrophic if users don't have data backed-up elsewhere. Instead of wiping the directory, could the toolbox be modified to create a new BIDS export folder in whatever output directory the user chooses? The new directory could have a timestamp appended in case there is more than one BIDS folder. I think this solution would be a better user interface and prevent accidental file deletion.

**Comment by effigies** [comment]
Sorry, what tool are you using?

**Reply by jacobsen-noelle** [reply]
I'm using bids-matlab-tools 7.3, my labmate is using 5.3

**Reply by effigies** [reply]
I would create an issue here: https://github.com/sccn/bids-matlab-tools/issues

This repository is for the BIDS specification itself.

**Reply by jacobsen-noelle** [reply]
Ah yes, thanks. I will do that.
